src/config.rs

```diff
@@ -257,14 +257,14 @@ mod tests {
         let tmp_dir = Utf8Path::from_path(tmp_dir.path()).unwrap();
         let config_dir = tmp_dir.join("lurk.toml");
         let public_params_dir = tmp_dir.join("public_params").into_string();
-        let perf_config = PerfConfig::from(CannedConfig::MaxParallelSimple);
+        let perf_config = PerfConfig::from(CannedConfig::FullySequential);
 
         let mut config_file = File::create(config_dir.clone()).unwrap();
         config_file
             .write_all(format!("public_params_dir = \"{public_params_dir}\"\n").as_bytes())
             .unwrap();
         config_file
-            .write_all("perf = \"max-parallel-simple\"\n".as_bytes())
+            .write_all("perf = \"fully-sequential\"\n".as_bytes())
```

Comment:
For the record, GitHub doesn't show it on my screen but this change is on a test called `test_config_override`

Reply:
Could you clarify? The change in question is for `test_config_lurk`, not for `test_config_override`

Reply:
Sorry, yeah.

I said that because those changes were unexpected to me at first. But they're on a test. So other reviewers can know this beforehand